src/test/java/com/google/api/codegen/util/py/PythonTypeTableTest.java

```diff
@@ -0,0 +1,46 @@
+/* Copyright 2017 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.util.py;
+
+import com.google.api.codegen.util.TypeAlias;
+import com.google.common.truth.Truth;
+import org.junit.Test;
+
+public class PythonTypeTableTest {
+
+  @Test
+  public void testGetAndSaveNicknameFor_disambiguate_movePackage() {
+    PythonTypeTable typeTable = new PythonTypeTable("foo.bar");
+    typeTable.getAndSaveNicknameFor(TypeAlias.create("a.c.D", "c.D"));
+    Truth.assertThat(typeTable.getAndSaveNicknameFor(TypeAlias.create("b.c.E", "c.E")))
```

Comment:
It does, but that isn't registered until after the second import is saved. I could check the `getImports` to make sure the changes are there.